runners/direct-java/src/main/java/org/apache/beam/runners/direct/ParDoMultiEvaluatorFactory.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Licensed to the Apache Software Foundation (ASF) under one
+* Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Will do..  Out of curiosity, is there a reason we don't have Checkstyle enforcing the header?  (we do in CXF)